workshop/exercise-2/README.md

```diff
@@ -220,10 +254,18 @@ The resulting directory structure of your Appsody project should look like this:
 └── quote.js
 ```
 
-Watch for the container to restart and then refresh your browser again. You will see a form appear.
+This is what the frontend looks like:
 
 ![Sample web form](images/screenshot.png)
 
+You can see the application by running the following:
+
+```bash
+echo $VM_IP:3000/quote
```

Comment:
also at http:// to command

Reply:
fixed in [this commit ](https://github.com/IBM/cloudpakforapps-workshop/pull/94/commits/11503d50fa71c8d0ddbf5c3b49e53811eab5918a)